site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -255,12 +262,24 @@ endif::includeRoles[]
 [field]#application.samlv2Configuration.audience# [type]#[String]# [optional]#Optional# [default]#defaults to `issuer`# [since]#Available since 1.6.0#::
 The audience for the SAML response sent to back to the service provider from FusionAuth. Some service providers require different audience values than the `issuer` and this configuration option lets you change the `audience` in the response.
 
-[field]#application.samlv2Configuration.callbackURL# [type]#[String]# [required]#Required# [since]#Available since 1.6.0#::
+[field]#application.samlv2Configuration.authorizedRedirectURLs# [type]#[Array<String>]# [required]#Required# [since]#Available since 1.20.0#::
+A list of URLs for the callback (sometimes called the Assertion Consumer Service or ACS). This is a set of values where FusionAuth may send the browser after the user logs in via SAML.
+
+[field]#application.samlv2Configuration.callbackURL# [type]#[String]# [optional]#Optional# [since]#Available since 1.6.0# [deprecated]#Deprecated#::
 The URL of the callback (sometimes called the Assertion Consumer Service or ACS). This is where FusionAuth sends the browser after the user logs in via SAML.
++
+[deprecated]#Deprecated in version 1.20.0# In version 1.20.0 and beyond, Callback URLs can be managed via [field]#application.samlv2Configuration.authorizedRedirectURLs#.
```

Comment:
This needs a line return after the deprecated block (I'll check in some CSS to fix the layout), example, formatting and updated text:

```
[deprecated]#Deprecated in version 1.20.0#
The callback URL is now managed using the [field]#samlv2Configuration.authorizedRedirectURLs# field. 
```

Reply:
Most usages don't have this from what I can tell. But I will add it.